### ISSUE TITLE
FLV video not working

### ISSUE THREAD

**Issue by Joppe27** [ppy/osu#8366]
FLV video playback is not working, it will not start and just shows the regular map background as if video was turned off. No black screen. 
version: latest (also doesn't work on 2020.221.0, even though it was fixed for some users ppy/osu#6099 )
logs: none

**Comment by bdach** [comment]
No logs?

**Comment by Joppe27** [comment]
> No logs?

No, I don't have any relevant logs to this issue. I should probably give you my hardware configuration though: Intel i5-7400, Intel HD 630, 8GB ram

**Comment by bdach** [comment]
To clarify, are there no logs at all, or do you not *think* they're relevant? If it's the second one, please humor me and attach them anyway.

As far as I'm concerned I cannot reproduce the problem. I don't think your hardware specs are very relevant here. I'd be more interested in the OS you're running.

**Comment by Joppe27** [comment]
> To clarify, are there no logs at all, or do you not _think_ they're relevant? If it's the second one, please humor me and attach them anyway.
> 
> As far as I'm concerned I cannot reproduce the problem. I don't think your hardware specs are very relevant here. I'd be more interested in the OS you're running.

I'll check again tomorrow and attach the logs too. I'm running windows 10

**Comment by peppy** [comment]
I’m more interested in the beatmap. This sounds like a parsing error

**Comment by Joppe27** [comment]
These are the logs: https://www81.zippyshare.com/v/6Kch3clT/file.html
These are the maps that don't work (the only FLV maps that I have): https://osu.ppy.sh/beatmapsets/459293#osu/984334
https://osu.ppy.sh/beatmapsets/10258#osu/40156
https://osu.ppy.sh/beatmapsets/10233#osu/40058
https://osu.ppy.sh/beatmapsets/10435#osu/41140

**Comment by bdach** [comment]
All four maps work fine here, no parsing errors. Nothing in the logs, either.

Are you sure that you don't just have video turned off in options?

**Comment by Joppe27** [comment]
> All four maps work fine here, no parsing errors. Nothing in the logs, either.
> 
> Are you sure that you don't just have video turned off in options?

I'm absolutely sure video was turned on. I would record a video but I don't have access to that pc right now. It's only FLV video that doesn't work on that pc, other formats are fine. FLV works fine on other computers too, it's just that specific computer that won't work with FLV so it's not a problem. Thanks for your help!

**Comment by huupoke12** [comment]
Did you download the non-video beatmaps? Can you try deleting those beatmaps and try redownloading them?

**Comment by Joppe27** [comment]
> Did you download the non-video beatmaps? Can you try deleting those beatmaps and try redownloading them?

Yup, they are video beatmaps. I thought this was the issue at first too so I redownloaded them to no avail.

**Comment by peppy** [comment]
@Joppe27 when you say "latest" could you please state your version?

**Comment by Joppe27** [comment]
> @Joppe27 when you say "latest" could you please state your version?

It's 2020.320.1

**Comment by Joppe27** [comment]
I figured it out, apparently osu!lazer doesn't check for any missing/changed (video) files when skipping imports because of a recently deleted beatmap. This caused the map to be restored instead of importing the new one with included video. I needed to delete the map, close out of osu!lazer, start it again and then drag the new one in, now the video does in fact work. I suggest adding a check to make sure the previously deleted beatmap is identical to the new one before skipping import and restoring, so things like this don't happen in the future. I found out about this because I changed the background on one of these maps on this computer, and when I imported the new map the custom background the I changed was still there. This is also logged in database, but I most likely overlooked it. Thanks again for helping everyone!
